roles/kargo-defaults/defaults/main.yaml

```diff
@@ -97,11 +97,15 @@ kube_apiserver_insecure_port: 8080 # (http)
 # Path used to store Docker data
 docker_daemon_graph: "/var/lib/docker"
 
+# Docker log options
+# Rotate container stderr/stdout logs at 50m and keep last 5
+docker_log_opts: "--log-opt max-size=50m --log-opt max-file=5"
```

Comment:
@holser I think it's already wrapped in the 'docker_log_opts'  variable
they may want also to use a different log_driver etc. 
So:
1. `docker_log_opts: ''`
2. `docker_logs_opts: "--log-opt max-size=2000m --log-opt max-file=2"`